### ISSUE TITLE
Masks is a list

### ISSUE THREAD

**Issue by tahreem-zaheer** [MouseLand/cellpose#337]
I am trying to obtain the value of the center of each cell, for this purpose I tried running the function  "def masks_to_flows_gpu(masks, device=None)" (which only requires masks). I passed the same value of mask that was generated by model.eval as the input parameter of this function requires, but I am getting an error that states masks is a list that does not have a shape. Which mask value do I have to pass to this? I am new to image processing and python so I would appreciate if anyone can help.

**Comment by Prehani** [comment]
Are you looking for the centroid of each cell? If so, you can run each mask in the list through scikit-image region props and pull out the centroid: https://scikit-image.org/docs/dev/api/skimage.measure.html#regionprops 

Should be something like this: 
```
masks, flows, styles, diams = model.eval(curr_im, diameter=0, flow_threshold=None, channels=channels, invert=invert_bool, gpu=gpu_bool)
from skimage.measure import regionprops
props = regionprops(masks)
centroids = props[0]['centroid']
```

**Comment by carsen-stringer** [comment]
thanks @Prehani ! yes this code will get you the cell centers
